dockerfiles/base-ci-linux/Dockerfile

```diff
@@ -22,6 +22,9 @@ WORKDIR /builds
 
 # install tools and dependencies
 COPY clang9.key /etc/apt/trusted.gpg.d/debian-archive-jessie-automatic.gpg
```

Comment:
I didn't see that before, but is that key for this http://apt.llvm.org/stretch/ repo from below. the name is misleading imo.
btw there would also be a debian package in the testing suite i.e. https://packages.debian.org/bullseye/clang-9

Reply:
The name of the key was suggested by @TriplEight  and I quite understand.
Questions, with the increase of the version from `stretch` to `buster` or from `xenial` to `bionic` I already raised in the channel devops.

Reply:
@gabreal , what name do you suggest? I thought it's pretty much straightforward, we install `clang-9` and `libclang-9-dev` from that repo.

Reply:
The name is misleading as the resulting filename `debian-archive-jessie-automatic.gpg` gives no clue that it is actually the key for the clang repo.

Reply:
Now, that you switched the base image to `buster`, I don't see why it would be needed to add llvms repository at all. IMO it's better to use the distro packages because they are supported by the distros' community and have been audited by a third-parity which is not the software vendor. But it's not so important and I can live with it if the key is properly verified, it's just another point where things can go wrong.